src/test/java/uk/gov/hmcts/reform/coh/controller/AnswerControllerTest.java

```diff
@@ -419,4 +456,10 @@ public void testUpdateAnswersInvalidAnswerState() throws Exception {
                 .andReturn();
         assertEquals("Answer state is not valid", result.getResponse().getContentAsString());
```

Comment:
This assertion fails as the controller includes the e.message in the response body too.eg. Answer state is not valid: answer_drafted

Reply:
Fixed